docs/regressions-msmarco-passage-unicoil-noexp.md

```diff
@@ -0,0 +1,140 @@
+```
+
+Alternatively, you can simply copy/paste from the commands below and obtain the same results.
+
+## Indexing
+
+Sample indexing command:
+
+```
+target/appassembler/bin/IndexCollection \
+  -collection JsonVectorCollection \
+  -input /path/to/msmarco-passage-unicoil-noexp \
+  -index indexes/lucene-index.msmarco-passage-unicoil-noexp/ \
+  -generator DefaultLuceneDocumentGenerator \
+  -threads 16 -impact -pretokenized \
+  >& logs/log.msmarco-passage-unicoil-noexp &
+```
+
+The path `/path/to/msmarco-passage-unicoil-noexp/` should point to the corpus downloaded above.
+
+The important indexing options to note here are `-impact -pretokenized`: the first tells Anserini not to encode BM25 doclengths into Lucene's norms (which is the default) and the second option says not to apply any additional tokenization on the uniCOIL tokens.
+Upon completion, we should have an index with 8,841,823 documents.
+
+For additional details, see explanation of [common indexing options](common-indexing-options.md).
+
+## Retrieval
+
+Topics and qrels are stored in [`src/main/resources/topics-and-qrels/`](../src/main/resources/topics-and-qrels/).
+The regression experiments here evaluate on the 6980 dev set questions; see [this page](experiments-msmarco-passage.md) for more details.
+
+After indexing has completed, you should be able to perform retrieval as follows:
+
+```
+target/appassembler/bin/SearchCollection \
+  -index indexes/lucene-index.msmarco-passage-unicoil-noexp/ \
+  -topics src/main/resources/topics-and-qrels/topics.msmarco-passage.dev-subset.unicoil-noexp.tsv.gz \
+  -topicreader TsvInt \
+  -output runs/run.msmarco-passage-unicoil-noexp.unicoil.topics.msmarco-passage.dev-subset.unicoil-noexp.txt \
+  -impact -pretokenized &
+```
+
+Evaluation can be performed using `trec_eval`:
+
+```
+tools/eval/trec_eval.9.0.4/trec_eval -c -m map -c -m recip_rank -c -m recall.1000 src/main/resources/topics-and-qrels/qrels.msmarco-passage.dev-subset.txt runs/run.msmarco-passage-unicoil-noexp.unicoil.topics.msmarco-passage.dev-subset.unicoil-noexp.txt
+```
+
+## Effectiveness
+
+With the above commands, you should be able to reproduce the following results:
+
+| MAP                                                                                                          | uniCOIL w/ doc2query-T5 expansion|
+|:-------------------------------------------------------------------------------------------------------------|-----------|
+| [MS MARCO Passage: Dev](https://github.com/microsoft/MSMARCO-Passage-Ranking)                                | 0.3215    |
+
+
+| MRR                                                                                                          | uniCOIL w/ doc2query-T5 expansion|
+|:-------------------------------------------------------------------------------------------------------------|-----------|
+| [MS MARCO Passage: Dev](https://github.com/microsoft/MSMARCO-Passage-Ranking)                                | 0.3263    |
```

Comment:
MRR for msmarco-passage should use -M 100 ?

Reply:
This is okay... this isn't the same as the MRR metric computed by the official MS MARCO script.  Should we just remove to reduce confusion?

MRR@k computed by `trec_eval` is not the same as the score computed by the official scripts...